src/common/core/core.js

```diff
@@ -3911,7 +3942,406 @@ define([
+         * @param {module:Core~Node} node - the node in question.
+         * @param {string} name - the name of the set in question.
+         *
+         * @return {module:Core~Node} The result is the closest base that defines the given aspect.
+         *
+         * @throws {CoreIllegalArgumentError} If some of the parameters doesn't match the input criteria.
+         * @throws {CoreIllegalOperationError} If the context of the operation is not allowed.
+         * @throws {CoreAssertError} If some internal error took place inside the core layers.
+         */
+        this.getAspectDefinitionOwner = function (node, name) {
+            ensureNode(node, 'node');
+            ensureType(name, 'name', 'string');
+
+            if (core.getValidAspectNames(node).indexOf(name) === -1) {
+                throw new CoreIllegalOperationError('Not valid aspect name [' + name + '] of the node.');
+            }
+
+            return core.getAspectDefinitionOwner(node, name);
+        };
+
+        /**
+         * Moves an own member of the set over to another set of the node.
+         * @param {module:Core~Node} node - the node in question.
+         * @param {string} memberPath - the path of the memberNode that should be moved.
+         * @param {string} oldSetName - the name of the set where the member is currently reside.
+         * @param {string} newSetName - the name of the target set where the member should be moved to.
+         *
+         * @throws {CoreIllegalArgumentError} If some of the parameters doesn't match the input criteria.
+         * @throws {CoreIllegalOperationError} If the context of the operation is not allowed.
+         * @throws {CoreAssertError} If some internal error took place inside the core layers.
+         */
+        this.moveMember = function (node, memberPath, oldSetName, newSetName) {
+            ensureNode(node, 'node');
+            ensureType(memberPath, 'memberPath', 'string');
+            ensurePath(memberPath, 'memberPath');
+            ensureType(oldSetName, 'oldSetName', 'string');
+            ensureType(newSetName, 'newSetName', 'string');
+
+            if (core.getSetNames(node).indexOf(oldSetName) === -1) {
+                throw new CoreIllegalOperationError('Source set [' + oldSetName + '] does not exists.');
+            }
+
+            if (core.getSetNames(node).indexOf(newSetName) === -1) {
+                throw new CoreIllegalOperationError('Target set [' + newSetName + '] does not exists.');
+            }
+
+            if (core.getOwnMemberPaths(node, oldSetName).indexOf(memberPath) === -1) {
+                throw new CoreIllegalOperationError('Not own member of the set therefore cannot be moved.');
+            }
+
+            return core.moveMember(node, memberPath, oldSetName, newSetName);
+        };
+
+        /**
+         * Renames the given attribute definition of the node. It also renames the default value of the definition!
```

Comment:
It also renames the default value of the definition, but does not rename any values stored at instances.